app/js/sagas/labOrdersSaga.js

```diff
@@ -187,9 +186,7 @@ function* updateOrders() {
   const labResultsTestOrderType = selectProperty(state, 'labResultsTestOrderType');
   const ordersBatchSize = selectProperty(state, 'ordersBatchSize');
   const options = {
-    dateToField: moment(labOrdersListFilters.dateToField).format('YYYY-MM-DD'),
-    dateFromField: moment(labOrdersListFilters.dateFromField).format('YYYY-MM-DD'),
-    excludeCanceledAndExpired: true,
+    ...labOrdersListFilters,
     ordersBatchSize: (ordersBatchSize || DEFAULT_ORDERS_BATCH_SIZE),
```

Comment:
I'm not sure why we were not passing in all the filters here, but the net affect was that after reloading after cancelled an order, it wouldn't use the proper filters on it's search... (I'm guessing this was old code we forgot to update after the filters were added?)